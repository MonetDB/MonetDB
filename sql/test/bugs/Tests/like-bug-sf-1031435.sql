start transaction;
create table like_test (str varchar(10));
insert into like_test values('');
insert into like_test values('t');
insert into like_test values('ts');
insert into like_test values('tsz');
select * from like_test;
select * from like_test where str like 't_';
rollback;
