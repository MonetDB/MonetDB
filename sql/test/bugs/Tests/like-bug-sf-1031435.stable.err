stderr of test 'like-bug-sf-1031435` in directory 'sql/test/bugs` itself:


# 14:39:27 >  
# 14:39:27 >  Mtimeout -timeout 180 Mserver "--config=/home/manegold/Monet/Testing/Current/.GNU.32.32.d.--enable-debug.PREFIX./etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/home/manegold/Monet/Testing/Current/.GNU.32.32.d.--enable-debug.PREFIX./lib/MonetDB:/home/manegold/Monet/Testing/Current/.GNU.32.32.d.--enable-debug.PREFIX./lib/bin" --set "gdk_dbfarm=/home/manegold/Monet/Testing/Current/.GNU.32.32.d.--enable-debug.PREFIX./var/MonetDB/dbfarm" --set "sql_logdir=/home/manegold/Monet/Testing/Current/.GNU.32.32.d.--enable-debug.PREFIX./var/MonetDB/log" --set mapi_port=32198 --set sql_port=40084 --set xquery_port=58857 --set monet_prompt= --trace "--dbname=mTests_src_test_bugs" --dbinit="module(sql_server); sql_server_start();" ; echo ; echo Over..
# 14:39:27 >  


# 14:39:27 >  
# 14:39:27 >  Mtimeout -timeout 60 MapiClient -lsql -u monetdb -P monetdb --host=localhost --port=40084  < like-bug-sf-1031435
# 14:39:27 >  


# 14:39:27 >  
# 14:39:27 >  Done.
# 14:39:27 >  

