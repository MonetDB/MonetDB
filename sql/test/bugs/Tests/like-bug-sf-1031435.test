statement ok
start transaction

statement ok
create table like_test (str varchar(10))

statement ok
insert into like_test values('')

statement ok
insert into like_test values('t')

statement ok
insert into like_test values('ts')

statement ok
insert into like_test values('tsz')

query T rowsort
select * from like_test
----
(empty)
t
ts
tsz

query T rowsort
select * from like_test where str like 't_'
----
ts

statement ok
rollback

