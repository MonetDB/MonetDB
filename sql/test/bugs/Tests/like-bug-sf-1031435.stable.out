stdout of test 'like-bug-sf-1031435` in directory 'sql/test/bugs` itself:


# 12:54:37 >  
# 12:54:37 >  Mtimeout -timeout 180 Mserver "--config=/ufs/fabian/scratch/monetdb/current/program-x86_64/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/fabian/scratch/monetdb/current/program-x86_64/lib/MonetDB:/ufs/fabian/scratch/monetdb/current/program-x86_64/lib/bin" --set "gdk_dbfarm=/ufs/fabian/scratch/monetdb/current/program-x86_64/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/fabian/scratch/monetdb/current/program-x86_64/var/MonetDB/log" --set mapi_port=38933 --set sql_port=45579 --set xquery_port=52530 --set monet_prompt= --trace "--dbname=mTests_src_test_bugs" --dbinit="module(sql_server); sql_server_start();" ; echo ; echo Over..
# 12:54:37 >  

# Monet Database Server V4.9.3
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.




# 12:54:45 >  
# 12:54:45 >  Mtimeout -timeout 60 MapiClient -lsql -u monetdb -P monetdb --host=localhost --port=45579  < like-bug-sf-1031435
# 12:54:45 >  

[ 1	]
[ 1	]
[ 1	]
[ 1	]
% sys.like_test # table_name
% str # name
% varchar # type
% 3 # length
[ ""	]
[ "t"	]
[ "ts"	]
[ "tsz"	]
% sys.like_test # table_name
% str # name
% varchar # type
% 2 # length
[ "ts"	]

# 12:54:45 >  
# 12:54:45 >  Done.
# 12:54:45 >  

